conversion/public/bite.v1.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+    self = this;
+
+    var actions = {
+        'init': function (programId) {
+            self.programId = programId;
+        },
+        'send': function (hitType, eventCategory, eventAction, eventLabel, eventValue) {
```

Comment:
Does not support the object parameter yet, all arguments has to be passed separately